spec/units/peep_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../setup_test_database.rb'
+
+describe Peep do
+
+  it 'shows all peeps' do
+    add_row_to_test_database
+    peeps = Peep.all
+    expect(peeps.first['message']).to eq 'This is a peep!'
+  end
+
+  it 'adds peeps' do
```

Comment:
This test does not seem to test that your application can add a peep to the database, but it seems you're rather testing that you can establish a connection manually with the db via PG and, after using the helper to add a row to the test db, retrieving all of the results. 

But this is something that happens in the previous test where you test the `.all` function from the `Peep` class if I'm not mistaken.